pact.cabal

```diff
@@ -316,8 +316,10 @@ library
         , hedgehog >= 1.0.1 && < 1.2
         , hspec
         , hspec-golden >= 0.1.0.2
+        , http-client
         , hw-hspec-hedgehog == 0.1.*
         , intervals
+        , temporary >= 1.3
```

Comment:
Did you mean to include these `pact.cabal` changes? I don't see any imports related to these new modules in this PR...

Reply:
this is for the `tests-in-lib` flag, it was missing from the test deps.